Doc/source/conf.py

```diff
@@ -118,8 +123,8 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
-html_style = 'default.css'
+html_theme = 'sphinxdoc'
+html_theme_options = { 'sidebarwidth': '350' }
```

Comment:
There was some particularly annoying reason why we needed this in dbprocessing. We can leave it out entirely (stick with the default 230) which isn't _too_ bad, or otherwise 300 looks to be sufficient. Or 20em.